test/e2e/framework/authorizer_util.go

```diff
@@ -62,7 +62,7 @@ func WaitForNamedAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviews
 		// GKE doesn't enable the SAR endpoint.  Without this endpoint, we cannot determine if the policy engine
 		// has adjusted as expected.  In this case, simply wait one second and hope it's up to date
 		if apierrors.IsNotFound(err) {
-			fmt.Printf("SubjectAccessReview endpoint is missing\n")
+			glog.Infof("SubjectAccessReview endpoint is missing\n")
```

Comment:
Since we did not need to format anything here, how about just `glog.Info()`?

Reply:
+1